servers/src/main/java/tachyon/web/WebInterfaceWorkerBlockInfoServlet.java

```diff
@@ -122,8 +125,6 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
         uiFileInfos.add(getUiFileInfo(tFS, fileId));
       }
       request.setAttribute("fileInfos", uiFileInfos);
```

Comment:
I think we check for `null` in the jsp, we should probably define a contract of which params are guaranteed to be set and which are optional